PROTOCOL.md

```diff
@@ -236,6 +236,20 @@ __Request fields__ :
 __Response__ : OK if source exists, with these additional fields :
 - **"name"** (string) : name of the requested source
 - **"volume"** (double) : volume of the requested source, on a linear scale (0.0 to 1.0)
+- **"muted"** (bool) : mute status of the requested source
+
+### "SetMute"
```

Comment:
The title is one level too high. Should be `#### "SetMute"` instead of `### "SetMute"`